modules/client-configuration/pages/vhm-caasp.adoc

```diff
@@ -59,7 +59,305 @@ Issuing those operations could render your {caasp} cluster unusable.
 {productname} will not stop you from issuing these operations if the system is not locked.
 ====
 
+== Autoinstallation profile of a {caasp}{nbsp}4 node
 
+{caasp}{nbsp}4 provides an AutoYaST profile to autoinstall a node in the `patterns-caasp-Management` package (see https://documentation.suse.com/suse-caasp/4.2/single-html/caasp-deployment/#_autoyast_preparation).
+The following is the {caasp}{nbsp}4 template customized to make use of {productname} channels and minion registration script that you can use as a starting point to automatically install a {caasp}{nbsp}4 using the Autoinstallation feature of {productname}.
```

Comment:
```suggestion
{caasp}{nbsp}4 provides an AutoYaST profile that you can use to autoinstall a node.
The profile is in the ``patterns-caasp-Management`` package.
For more information about the profile, see https://documentation.suse.com/suse-caasp/4.2/single-html/caasp-deployment/#_autoyast_preparation.

The template in this example is based on the {caasp}{nbsp}4 template, customized to make use of {productname}.
It includes channels and client registration scripts to install a {caasp}{nbsp}4 cluster with {productname} autoinstallation.
This script is intended for you to use as a basis for your own autoinstallation scripts.
```

Reply:
Yes, I know I made it longer. You had a lot of information packed into those two sentences!